src/app.ts

```diff
@@ -27,7 +27,7 @@ void (async () => {
   console.log('Checked post contracts')
 
   console.log('Starting contract listeners...')
-  await startContractListeners()
+  startContractListeners()
```

Comment:
Why did you remove `await` from here?

Reply:
We don't need, the function is not asynchronous